mne/realtime/fieldtrip_client.py

```diff
@@ -171,6 +171,8 @@ def _guess_measurement_info(self):
                     this_info['kind'] = FIFF.FIFFV_ECG_CH
                 elif ch.startswith('MISC'):
                     this_info['kind'] = FIFF.FIFFV_MISC_CH
+                elif ch.startswith('SYS'):
+                    this_info['kind'] = FIFF.FIFFV_SYST_CH
```

Comment:
@jjnurminen this is cool. However, the easy way to get around this is to give `FieldTripClient` a measurement info. You can do:

```py
info = read_info('short_recording.fif')
rt_client = FieldTripClient(host='sinuhe', port=1972, info=info, ...)
```
I think in retrospect trying to guess the info like this was probably a bad idea ...